spec/changelog.yml

```diff
@@ -2,11 +2,11 @@
 ## This file documents changes in the package specification. It is NOT a package specification file.
 ## Newer entries go at the bottom of each in-development version.
 ##
-- version: 2.5.2-next
+- version: 2.5.2
```

Comment:
```suggestion
- version: 2.6.0-next
```

Reply:
thanks! I wasn't sure of the release process